rir/src/compiler/translations/pir_2_rir/pir_2_rir.cpp

```diff
@@ -851,8 +851,6 @@ rir::Code* Pir2Rir::compileCode(Context& ctx, Code* code) {
 
                 auto loadArg = [&](Value* what, size_t argNumber) {
                     if (what == UnboundValue::instance()) {
-                        assert(MkArg::Cast(instr) &&
```

Comment:
Is this change adding more possibilities that unbound values on phis? If that is the case I would prefer to keep the assert with an or including phis.

Reply:
phis, pircopy and mkenv, from the top of my head. do you want an assert on those?